n8n/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       PROXY_AUTH_WHITELIST: "/webhook-test/*,/webhook/*"
 
   server:
-    image: n8nio/n8n:0.234.1@sha256:b00e39ced253d0a5d6f234a21e7d8991dc795a99f13f32db479448087a14ff5a
+    image: n8nio/n8n:1.4.0@sha256:04796ef0bf9ea2b34135ef4f304dad853489bec7b26a3726d3f97df13175bf5c
```

Comment:
I have changed this digest to the available multi-arch image digest so that it works for Umbrel users on both arm64 and x86 architectures. When an image on Docker Hub is multi-arch, you can get the correct digest by running `docker pull <image>:<tag>`. Here is n8n 1.4.0 as an example:

```
$ sudo docker pull n8nio/n8n:1.4.0
1.4.0: Pulling from n8nio/n8n
...
Digest: sha256:04796ef0bf9ea2b34135ef4f304dad853489bec7b26a3726d3f97df13175bf5c
Status: Downloaded newer image for n8nio/n8n:1.4.0
docker.io/n8nio/n8n:1.4.0
```